src/main/java/exnihilocreatio/registries/registries/CrucibleRegistry.java

```diff
@@ -35,8 +38,6 @@ public CrucibleRegistry(List<? extends IDefaultRecipeProvider> defaultRecipeProv
         super(
                 new GsonBuilder()
                         .setPrettyPrinting()
-                        .registerTypeAdapter(ItemInfo.class, new CustomItemInfoJson())
```

Comment:
Same here, Meltable still has it.

Reply:
Meltable was changed to use BlockInfo instead of ItemInfo. I'll re-add it in the form of BlockInfo

Reply:
That might break with the new intended syntax as I did only change the ItemInfoJson syntax but not the BlockInfo

Reply:
It didn't make much sense to use ItemInfo before as the result is always a block, so I thought for clarity changing it to BlockInfo would help